admission-webhook/deploy/deploy-gmsa-webhook.sh

```diff
@@ -104,7 +104,7 @@ main() {
     CREATE_SIGNED_CERT_SCRIPT=$(ensure_helper_file_present 'create-signed-cert.sh')
 
     if [ ! -x "$(command -v envsubst)" ]; then
-        fatal_error 'envsubst not found'
+        apt-get update && apt-get install -y --no-install-recommends gettext-base || fatal_error 'Failed to install envsubst'
```

Comment:
The `apt-get` may be wrapped in a distro check (to make sure RHEL/etc. that do not use `apt` simply errors out like earlier).

Reply:
There are a number of (reasonablish) things that this script expects to have present prior to running; it's the admin's job to ensure that they are.

That being said, `envsubst` might indeed be a bit of a stretch compared to the other required tools. To avoid getting into distro checks, how about using something like https://hub.docker.com/r/dibi/envsubst if `envsubst` is not present?